validator/validator-main.protoascii

```diff
@@ -313,6 +313,39 @@ tags: {
   attr_lists: "common-link-attrs"
   spec_url: "https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#html-tags"
 }
+# Allowlisted CSS provider
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "LINK"
+  spec_name: "link rel=stylesheet for amp-story-1.0 css"
+  descriptive_name: "link rel=stylesheet for amp-story-1.0 css"
+  mandatory_parent: "HEAD"
+  attr_lists: "nonce-attr"
+  attrs: { name: "crossorigin" }  # SRI attribute (https://www.w3.org/TR/SRI/)
+  attrs: {
+    name: "href"
+    mandatory: true
+    value: "https://cdn.ampproject.org/v0/amp-story-1.0.css"
+    value: "https://cdn.ampproject.org/lts/v0/amp-story-1.0.css"
+  }
+  attrs: { name: "integrity" }  # SRI attribute (https://www.w3.org/TR/SRI/)
+  attrs: { name: "media" }
+  attrs: {
+    name: "rel"
+    mandatory: true
+    value_casei: "stylesheet"
+    dispatch_key: NAME_VALUE_DISPATCH
+  }
+  attrs: {
+    name: "type"
+    value_casei: "text/css"
+  }
+  attrs: {
```

Comment:
@Gregable @jridgewell i've added to allow `amp-extension=amp-story` for this `link`. This will make it easier to detect/query the extension external stylesheet.

i'll make the changes to the amp cache transformer and toolbox optimizer if y'all think this is OK

Reply:
I think that's appropriate.